packages/webapp/src/inductions/components/induction-lists/invitee-inductions.tsx

```diff
@@ -47,39 +42,23 @@ const INVITEE_INDUCTION_COLUMNS: InductionTable.Column[] = [
 
 const getTableData = (inductions: Induction[]): InductionTable.Row[] => {
     return inductions.map((induction) => {
-        const { data: allEndorsements } = useQuery(
-            queryEndorsementsByInductionId(induction.id)
-        );
-
         const { data: inviter } = useMemberByAccountName(induction.inviter);
```

Comment:
This hook is still inside the map. Are we wanting to get rid of that one too?

Reply:
nice catch! Working on it!